src/main/java/bio/terra/common/ValidationUtils.java

```diff
@@ -49,4 +50,12 @@ public static boolean isValidUuid(String uuid) {
       return Optional.empty();
     }
   }
+
+  public static String requiresNotBlank(String value, String errorMsg) {
```

Comment:
~I don't think this is necessary, you can replace its use with `Objects.requireNonNull()`.~ Nevermind, this is checking for empty as well as `null`. I would expect the method to be `require` not `requires` though.
```suggestion
  public static String requireNotBlank(String value, String errorMsg) {
```

Reply:
I don't think `Objects.requireNonNull` provide the validation I want -  which is `StrigUtils.isNotEmpty()`, Null, Empty or Blank .

Reply:
I agree with Jesus on this one. This is a little more than just a null check

Reply:
Yeah I updated my comment once I re-read the code. The only note left is to change the name to `requireNotBlank()` since our other methods that do similar things are `require` not `requires`.

Reply:
Renamed.